bitcoin/src/taproot.rs

```diff
@@ -1064,15 +1062,17 @@ impl fmt::Display for TaprootError {
                 "Merkle Tree depth({}) must be less than {}",
                 d, TAPROOT_CONTROL_MAX_NODE_COUNT
             ),
-            TaprootError::InvalidTaprootLeafVersion(v) =>
-                write!(f, "Leaf version({}) must have the least significant bit 0", v),
+            TaprootError::InvalidTaprootLeafVersion(v) => {
```

Comment:
A couple formatting changes got sneaked in here. Don't think it should be an issue